FogCreek/Glitch-Community
# Scrawny puffin 1: fix search results storybook

## Links
* Remix link https://scrawny-puffin-1.glitch.me
* Manuscript link https://glitch.manuscript.com/f/cases/3328368/

## GIF/Screenshots:
![Screen Shot 2019-04-23 at 11 26 27 AM](https://user-images.githubusercontent.com/938722/56594349-ababdb80-65ba-11e9-8b74-76af02220ed5.png)

## Changes:
* add `searchResults.starterKit` to storybook and proptypes
* move component into own directory

## How To Test:
* visit https://scrawny-puffin-1.glitch.me/storybook/index.html?path=/story/searchresults--results
* the storybook should load
